### ISSUE TITLE
String out of bound exception

### ISSUE THREAD

**Issue by visze** [charite/jannovar#176]
Position 19:g.45010213:A>G

Jannovar release 0.14

RefSeq database

```
Exception in thread "main" java.lang.StringIndexOutOfBoundsException: String index out of range: 1495
        at java.lang.String.substring(String.java:1951)
        at de.charite.compbio.jannovar.reference.TranscriptSequenceDecorator.getCodonAt(TranscriptSequenceDecorator.java:85)   
        at de.charite.compbio.jannovar.annotation.builders.SNVAnnotationBuilder.buildCDSExonicAnnotation(SNVAnnotationBuilder.java:100)
        at de.charite.compbio.jannovar.annotation.builders.SNVAnnotationBuilder.build(SNVAnnotationBuilder.java:66)
        at de.charite.compbio.jannovar.annotation.builders.AnnotationBuilderDispatcher.build(AnnotationBuilderDispatcher.java:44)
        at de.charite.compbio.jannovar.annotation.VariantAnnotator.buildNonSVAnnotation(VariantAnnotator.java:168)
        at de.charite.compbio.jannovar.annotation.VariantAnnotator.buildAnnotations(VariantAnnotator.java:151)

```

**Comment by visze** [comment]
Same with annotate pos:

```
#java -jar jannovar-cli-0.14.jar annotate-pos data/hg19_refseq.ser 'chr19:45010213A>G'
Options
verbosity: 1
dataFile: data/hg19_refseq.ser
vcfFilePaths: []
chromosomalChanges: [chr19:45010213A>G]
showAll: false
jannovarFormat: false
writeJannovarInfoFields: false
writeVCFAnnotationStandardInfoFields: true
output infix: .jv
Deserializing transcripts...
Deserializing JannovarData from data/hg19_refseq.ser
Deserialization took 6.32 sec.
#change effect  hgvs_annotation
[ERROR] Could not annotate variant chr19:45010213A>G!
java.lang.StringIndexOutOfBoundsException: String index out of range: 1495
        at java.lang.String.substring(String.java:1951)
        at de.charite.compbio.jannovar.reference.TranscriptSequenceDecorator.getCodonAt(TranscriptSequenceDecorator.java:85)
        at de.charite.compbio.jannovar.annotation.builders.SNVAnnotationBuilder.buildCDSExonicAnnotation(SNVAnnotationBuilder.java:100)
        at de.charite.compbio.jannovar.annotation.builders.SNVAnnotationBuilder.build(SNVAnnotationBuilder.java:66)
        at de.charite.compbio.jannovar.annotation.builders.AnnotationBuilderDispatcher.build(AnnotationBuilderDispatcher.java:44)
        at de.charite.compbio.jannovar.annotation.VariantAnnotator.buildNonSVAnnotation(VariantAnnotator.java:168)
        at de.charite.compbio.jannovar.annotation.VariantAnnotator.buildAnnotations(VariantAnnotator.java:151)
        at de.charite.compbio.jannovar.cmd.annotate_pos.AnnotatePositionCommand.run(AnnotatePositionCommand.java:64)
        at de.charite.compbio.jannovar.Jannovar.main(Jannovar.java:104)

```

**Comment by visze** [comment]
problem does not appear with ucsc

**Comment by visze** [comment]
I found a problem with my program. GenomePosition in Jannovar is zero based (ups!!!). So the variant is 'chr19:45010212A>G' But with the same result: String out of bound exception

**Comment by holtgrewe** [comment]
I think also resolved by #210. Please check.

```
$ java -jar jannovar-cli/target/jannovar-cli-0.17-SNAPSHOT.jar annotate-pos -d data/hg19_refseq.ser -c 'chr19:45010212A>G'
ERROR StatusLogger No log4j2 configuration file found. Using default configuration: logging only errors to the console.
Options
verbosity: 1
dataFile: data/hg19_refseq.ser
vcfFilePaths: []
chromosomalChanges: [chr19:45010212A>G]
showAll: false
jannovarFormat: false
writeJannovarInfoFields: false
writeVCFAnnotationStandardInfoFields: true
output infix: .jv
Deserializing transcripts...
INFO Deserializing JannovarData from data/hg19_refseq.ser
INFO Deserialization took 4.77 sec.
#change	effect	hgvs_annotation
chr19:45010212A>G	THREE_PRIME_UTR_VARIANT,THREE_PRIME_UTR_VARIANT,THREE_PRIME_UTR_VARIANT,THREE_PRIME_UTR_VARIANT	CEACAM20:NM_001102597.1:c.*1T>C:p.(=),CEACAM20:NM_001102598.1:c.*1T>C:p.(=),CEACAM20:NM_001102599.1:c.*1T>C:p.(=),CEACAM20:NM_001102600.1:c.*1T>C:p.(=)
```
